### PR TITLE
API-12351: fix omitempty on struct fields[v3.4]

### PR DESCRIPTION
**Pull request by kacperf531**
https://livechatinc.atlassian.net/browse/API-12351

### CODE REVIEW

File: webhooks/structures.go [hunk 1]
```diff
@@ -252,7 +252,7 @@ type BotCreated struct {
 	MaxChatsCount        *uint                        `json:"max_chats_count,omitempty"`
 	DefaultGroupPriority configuration.GroupPriority  `json:"default_group_priority,omitempty"`
 	Groups               []*configuration.GroupConfig `json:"groups,omitempty"`
-	WorkScheduler        configuration.WorkScheduler  `json:"work_scheduler,omitempty"`
+	WorkScheduler        *configuration.WorkScheduler `json:"work_scheduler,omitempty"`
```
Comment (wfabjanczuk) [inline webhooks/structures.go]:
```suggestion
	WorkScheduler        configuration.WorkScheduler `json:"work_scheduler,omitempty"`
```
In v3.4 `configuration.WorkScheduler` is a map, which is a pointer field already so there's no need to use pointer here

File: webhooks/structures.go [hunk 2]
```diff
@@ -266,7 +266,7 @@ type BotUpdated struct {
...
```
Comment (wfabjanczuk) [inline webhooks/structures.go]:
```suggestion
	WorkScheduler        configuration.WorkScheduler `json:"work_scheduler,omitempty"`
```
Same as above

**Review by wfabjanczuk — CHANGES_REQUESTED** [review]

File: webhooks/structures.go (hunk 1 above)
Comment (kacperf531) [reply webhooks/structures.go]: fixed in [API-12351: apply review suggestions](https://github.com/livechat/lc-sdk-go/pull/144/commits/3bdbd1fbf9fad7510a48c7d8f26b5d6531691e32)

File: webhooks/structures.go (hunk 2 above)
Comment (kacperf531) [reply webhooks/structures.go]: fixed in [API-12351: apply review suggestions](https://github.com/livechat/lc-sdk-go/pull/144/commits/3bdbd1fbf9fad7510a48c7d8f26b5d6531691e32)

**Review by wfabjanczuk — APPROVED** [review]

**Pull request merged**